lib/Fhp/BaseAction.php

```diff
@@ -225,4 +230,20 @@ public function getRequestSegmentNumbers(): array
     {
         $this->tanRequest = $tanRequest;
     }
+
+    final public function setMessage(MessageBuilder $message)
+    {
+        $this->message = $message;
+    }
+
+    final public function getMessage()
+    {
+        if ($this->getPaginationToken() !== null && $this->message instanceof MessageBuilder) {
```

Comment:
Die Dokumentation bei `getPaginationToken()` trifft jetzt nicht mehr zu. Vielleicht können wir den Getter auch ganz loswerden? Wie wäre es z.B. wenn wir den Getter löschen und statt `$this->paginationToken = ...` unten einfach direkt in `$this->message` schreiben? Falls auf der "vorherigen Seite" ein Fehler aufgetreten sein sollte, wäre der zu dem Zeitpunkt ja schon geworfen worden, also dürfte es dann nicht schaden, die `$this->message` zu ändern.

Reply:
Ich hab mich erstmal dafür entschieden, den aufsetzpunkt erst in getNextRequestMessage zu setzen. Das scheint mir logischer zu sein, die message nicht vorher zu verändern.